catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitorIT.java

```diff
@@ -179,10 +180,12 @@ public void setup() throws Exception {
   }
 
   @Test
+  @Ignore // Ignoring for now as the test requires a security manager
```

Comment:
Is this going to be addressed in this pull request?

Reply:
No. It sounds like it would be a significant effort to enable the security needed for this test now. It is a simple create test that has been throughly tested manually though. If this is ported to a newer branch it would be worth looking into re-enabling it.